src/single-spa-angularjs.js

```diff
@@ -56,11 +50,12 @@ function mount(opts, mountedInstances, props = {}) {
   return Promise.resolve().then(() => {
     window.angular = opts.angular;
 
-    const containerEl = getContainerEl(opts, props);
+    const domElementGetter = chooseDomElementGetter(opts, props);
```

Comment:
I have changed the naming convention of `containerEl` and declaration flow to match the React version 😄